README.md

```diff
@@ -26,11 +26,23 @@ $ cd helerm-ui
 
 #### 2. Install depencies:
 
+1. Running the app locally
```

Comment:
I think the indentations / the order of steps looks a bit funny 🤔 Would suggest something like this:

### Install from source

Make sure you have [Yarn](https://yarnpkg.com/en/docs/install) installed globally.

#### 1. Clone the project:

```bash
$ git clone https://github.com/City-of-Helsinki/helerm-ui.git
$ cd helerm-ui
```

#### 2. Install depencies:

```bash
$ yarn              # Install project dependencies
```

#### 3. Running the app locally
```bash
$ yarn start:dev    # Launch on local environment
```

### Running the app in docker

#### 1. Add needed environmental variables
Add these by copying the `.env.example` to `.env` and adding the values to the file, or directly through process.env.

Requires you to setup a port where the application will be running to your .env file, f.ex.
```bash
PORT=3000
```

```bash
$ docker-compose up --build # Launches the containerized version of the application
```

You will need to have the CLIENT_* and JWT_TOKEN parameters for the Helsinki Tunnistamo, or **run your own https://github.com/City-of-Helsinki/tunnistamo instance**.

#### 2. Navigate to [http://localhost:3000](http://localhost:3000)